CoEpi/ui/HealthQuiz/HealthQuizViewModel.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  HealthQuizViewModel.swift
+//  CoEpi
+//
+//  Created by Johnson Hsieh on 3/26/20.
+//  Copyright © 2020 org.coepi. All rights reserved.
+//
+
+class HealthQuizViewModel {
+    weak var delegate: HealthQuizViewModelDelegate?
+
+    private var questions: Array<Question> = [
```

Comment:
You could merge https://github.com/Co-Epi/app-ios/pull/10
But we can do it in a follow up too.

In any case, you need to map the repo's symptoms to view data items, which would add the `checked` flag.